MediaManager/MediaManagerBase.cs

```diff
@@ -144,6 +145,15 @@ public virtual async Task<IMediaItem> Play(string uri)
             return mediaItem;
         }
 
+        public virtual async Task<IMediaItem> Play(string resourceName, Assembly assembly)
```

Comment:
Assembly assembly = null

Reply:
Added `TryFindAssembly` in `MediaExtractorBase`